### PR TITLE
add changelog

### PR DESCRIPTION
**Pull request by volllly**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+### Added
+
+- The fiskaltrust.Launcher is now open source.
+- Json launcher configuration file support.
+- Native hosting of GRPC urls.
+- Package download cache.
+- Handling of encrypted configuration values.
+- .NET 6 support.
+- Native executables for Windows, Linux and MacOS.
+- Lunix service installation.
```
Comment (TSchmiedlechner) [inline CHANGELOG.md:20]:
```suggestion
- Windows service installation.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+### Changed
+
+- Better console and file logging.
+- Better command line interface.
+
+### Removed
+
+- Support for `fiskaltrust.exe.config` files removed.
+- Support for .NET Framework 4.
```
Comment (TSchmiedlechner) [inline CHANGELOG.md:30]:
```suggestion
- Support for .NET Framework 4. (required framework is now included)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- Hosting of WCF/SOAP. (SOAP support will be re-added in a leter version)
```
Comment (TSchmiedlechner) [inline CHANGELOG.md:31]:
```suggestion
- Hosting of WCF/SOAP. (SOAP support will be re-added in a later version)
```

File: CHANGELOG.md (hunk 1 above)
Comment (volllly) [reply CHANGELOG.md:20]: I did not include windows service installation because It already exists in the legacy launcher 🤔

**Pull request merged**